src/plugins/hiddenRows/test/plugins/mergeCells.e2e.js

```diff
@@ -0,0 +1,348 @@
+describe('HiddenRows', () => {
```

Comment:
Shouldn't those tests be moved to `mergeCells` plugin and use `indexMapper.register(HiddenMap)` to make sure that `mergeCells` are working with any plugin that is hiding rows/columns, not only hiddenRows plugin?

Reply:
Moved the task to the separate issue #6908.